src/cell.cpp

```diff
@@ -56,7 +90,10 @@ Cell::Cell() {
 	// Instantiate the random number generator
 	_rand.instantiate();
 
-	//temp setup a genome for testing.
+	// Instantiate Genome
+	_cellGenome = Genome();
```

Comment:
Not necessary as the `Genome _cellGenome;` definition from the header will instantiate an instance of the genome class. It's only done like this inside seteq in order to deconstruct/overwrite the genes added by the default constructor.